keps/sig-node/1977-container-notifier/README.md

```diff
@@ -246,48 +246,51 @@ type PodNotificationStatus struct {
     // If set, it suggests the timestamp(UTC) at when the PodNotification has
     // been marked as completed. A PodNotification is completed when the
     // action has been executed in all eligible Containers in this Pod.
+    // Note that a completed PodNotification does NOT mean the notification
+    // has been executed successfully. Refer to `Succeeded` field for more details.
     // +optional
     CompleteTime *metav1.Time
 
 
     // Succeeded will be set to true when the action has been executed
-    // successfully in all matched containers within this Pod,
+    // successfully in all qualified containers within this Pod,
     // and to false otherwise. It will not be specified if the status of
-    // of execution is unknown(i.e., in execution).
+    // of execution is unknown(i.e., still in execution).
     // +optional
-    Succeeded *metav1.Time
+    Succeeded *bool
```

Comment:
I think this is better served by an enum `result` (or something like that) with values for success and failure, so we can later add other things.

Thoughts?

Reply:
make sense, will update